docs/user_documentation/admin-features/guide-ref-security.md

```diff
@@ -0,0 +1,102 @@
+# Setup security in MOLGENIS
+To understand the security context of MOLGENIS you need to understand the following concepts.
+
+* **Group**
+  
+  A number of people that work together.
+
+* **User**
+  
+  A person that can login to MOLGENIS.
+
+* **Roles**
+  
+  A role is a job function within the context of a group with certain permissions on resources. Examples of roles are:
+  * Managers
+  * Editors
+  * Viewers
+ 
+* **Resource**
+  
+  Resources can be tables or folders.
+
+* **Plugin**
+  
+  Plugins are specific modules within MOLGENIS that offer certain functionality. For example the [Data Explorer](../finding-data/guide-explore.md).
+      
+## Add groups
+*note: you need to be superuser to add groups*
+
+You can add groups in MOLGENIS by navigating to **Admin --> Security Manager**. 
+
+![Security Manager menu](../../images/security/main_menu_security.png?raw=true, "Security Manager menu")
```

Comment:
Remove the Chrome bar from this screenshot, it contains your name :wink:.
Screenshot bonus points: hover over 'Security Manager' in the dropdown menu for maximum clarity.